terraform/output.tf

```diff
@@ -0,0 +1,9 @@
+output "cluster_location" {
+    description = "Location of the cluster"
+    value = resource.google_container_cluster.cluster.location
+}
+
+output "cluster_name" {
+    description = "Name of the cluster"
```

Comment:
(nit) `GKE Cluster name:`
Here it's good to be succinct because this will be printed out at the end of the Terraform run